doc/doxygen/install/install-win.doxygen

```diff
@@ -313,6 +315,33 @@
     build OpenMS r
     \endcode
 
+  @section install_win_nonvs Advanced: Building OpenMS in any IDE (Example: Visual Studio Code)
+    
+    This section is for users that already have a little experience with IDEs like VS Code or JetBrains CLion.
+    Especially the first step requires to read through the beginning of this documentation.
+    <ol>
+      <li> Download and install requirements (Git, CMake, 7zip, VS Build Tools, Qt) and make sure they are in your PATH
+      <li> Download and install VS Code
+      <li> Install C++ Extension Pack, CMake Tools, CMake Test Explorer and potentially some GitHub extensions
+      <li> Ctrl+Shift+P and find <code>Git: clone</code>, then type and search for OpenMS/OpenMS. Select a folder. Wait for clone to finish. Open clone. Trust authors.
+      <li> Open bash Terminal from within VS Code and type <code>source tools/loadvcvarsall.sh && load_msenv && git submodule update --init contrib && mkdir contrib-build && cd contrib-build && cmake -DBUILD_TYPE=ALL ../contrib</code>
+      <li> Adapt (to your paths) and add
+        \code
+        "cmake.configureArgs": [
+          "-DOPENMS_CONTRIB_LIBS=C:\\git\\contrib-build",
+          "-DCMAKE_PREFIX_PATH=C:\\dev\\Qt\\5.15.2\\msvc2019_64"
+        ]
+        \endcode
+        to your settings or configure manually.
```

Comment:
Where would I put this in VS Code for example?

Reply:
oh in your user settings